src/components/layout/DashboardLayout.tsx

```diff
@@ -0,0 +1,54 @@
+import { useTranslation } from 'next-i18next'
+import { makeStyles } from '@mui/styles'
+import { observer } from 'mobx-react'
+
+import Snackbar from 'components/layout/Snackbar'
+
+import DashboardMiniDrawer from './DashboardMiniDrawer'
+import { DrawerStore } from 'stores/DrawerStore'
+
+type Props = {
+  children: React.ReactNode
+}
+
+const useStyles = makeStyles(() => {
+  return {
+    pageContainer: {
+      position: 'relative',
+      minHeight: '100vh',
+    },
+    contentWrapper: {
+      paddingBottom: '40px',
+    },
+    footer: {
+      backgroundColor: '#294E85',
+      position: 'absolute',
+      color: '#fff',
+      bottom: 0,
+      padding: '16px 20px',
+      width: '100%',
+      height: '40px',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'flex-end',
+    },
+  }
+})
+
+export default observer(function DashboardLayout({ children }: Props) {
```

Comment:
We already have a common component `AdminLayout` and we need to tweak this PR to use it